src/Popover/Popover.tsx

```diff
@@ -194,7 +264,7 @@ export default class Popover extends React.Component {
     return { top, left };
   };
 
-  saveValuesFromNodeTree = (cb) => {
+  saveValuesFromNodeTree = (cb?: () => any) => {
```

Comment:
actually `cb` is never passed to `saveValuesFromNodeTree` (it was until some months ago)...
I vote for removing it completely